include/libssh2.h

```diff
@@ -129,7 +129,7 @@ typedef unsigned char uint8_t;
 typedef unsigned int uint32_t;
 typedef unsigned __int64 libssh2_uint64_t;
 typedef __int64 libssh2_int64_t;
-#ifndef ssize_t
+#if !defined HAVE_SSIZE_T && !defined ssize_t
```

Comment:
we use parenthesis for all other uses of defined() in preprocessor lines so I think this should too!